apps/wpcom-block-editor/src/common/upgrade-nudge.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
This is essentially a duplicate of https://wpcalypso.wordpress.com/devdocs/blocks/upgrade-nudge